### PR TITLE
Change download to  2.1.1

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

**Comment by Pegasus-RPG** [comment]
LGTM. Have you tested that none of the links are broken?

**Comment by daschuer** [comment]
Yes.

File: pages/download.html [hunk 1]
```diff
@@ -16,20 +16,20 @@
 unstable_ubuntu64_analytics_conversion: /downloads/2.1.0-rc1-ubuntu64
 unstable_ubuntu_ppa_analytics_conversion: /downloads/2.1.0-rc1-ubuntu-ppa
 stable_short_version: Mixxx 2.1
```
Comment (Be-ing) [inline pages/download.html]: This needs to be updated to 2.1.1 too.

Comment (daschuer) [reply pages/download.html]: Why? It is still a 2.1 Version. We have also not had the minor version before.

Comment (Pegasus-RPG) [reply pages/download.html]: We should have had the full version number before!

Comment (Be-ing) [reply pages/download.html]: It is inconsistent to have 2.1 in some places but 2.1.1 in others.

**Comment by daschuer** [comment]
Done. Looks now less pleasant to me, but OK.  
http://staging.mixxx.org/download/

**Pull request merged**